From e6cfd29e7ce75b996e8e058e397cf237a1276643 Mon Sep 17 00:00:00 2001
From: Jonas Larsson <ljonas@google.com>
Date: Mon, 19 Aug 2019 11:21:30 -0700
Subject: [PATCH 1/1] vpudec: Remove log statement that segfaults

When enabling debug logs this log statement segfaults. Remove it.

Change-Id: I2c56cb09ec4b6bab94e3b4378a8838760fbb1aff
---
 plugins/vpu/gstvpu.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/plugins/vpu/gstvpu.c b/plugins/vpu/gstvpu.c
index 0526acd..c2454bc 100644
--- a/plugins/vpu/gstvpu.c
+++ b/plugins/vpu/gstvpu.c
@@ -68,9 +68,6 @@ gst_vpu_allocate_internal_mem (VpuInternalMem * vpu_internal_mem)
 	for (i = 0; i < vpu_internal_mem->mem_info.nSubBlockNum; ++i) {
 		size = vpu_internal_mem->mem_info.MemSubBlock[i].nAlignment \
            + vpu_internal_mem->mem_info.MemSubBlock[i].nSize;
-		GST_DEBUG_OBJECT(vpu_internal_mem, "sub block %d  type: %s  size: %d", i, \
-        (vpu_internal_mem->mem_info.MemSubBlock[i].MemType == VPU_MEM_VIRT) ? \
-        "virtual" : "phys", size);
  
 		if (vpu_internal_mem->mem_info.MemSubBlock[i].MemType == VPU_MEM_VIRT) {
       ptr = g_malloc(size);
-- 
2.26.0

